### ISSUE TITLE
Homebridge not showing in iOS app - maybe an Apple bug?

### ISSUE THREAD

**Issue by hawkeye217** [nfarina/homebridge#1525]
A couple of weeks ago, I began to experiment with Homebridge and a few accessories.  Installed everything, set up my config.json, all was working great.  

Got busy and didn't play around with much again until yesterday.  No changes to the server/environment where Homebridge runs.  I opened up Home on my iOS 10 iPhone, and got the dreaded "not responding" messages for everything.  

I went through all the steps lots of you already have - cleaned .homebridge/persist and .homebridge/accessories, rebooted everything, removed "My Home" altogether on the app... Homebridge would now no longer show up as an available accessory.  :(

So, I changed my username in config.json, and voila - Homebridge showed up!  I type in the code, and almost immediately after, I receive an "unable to add accessory" error message.

From that point, the Homebridge accessory never shows up on the app until I change the username in config.json again.  I can't get past that step on iOS anymore.

I'm wondering if anyone else has this issue.  I'm beginning to think it's something Apple has done.  Any insights?

**Comment by jclarke0000** [comment]
Yeah everything is showing up as "No Response" in the iOS Home app for me too.  I'm on iOS 11, so it doesn't appear to be iOS version related.

**Comment by 13mralex** [comment]
Refer to issue #1520, hopefully it will help.

**Comment by hawkeye217** [comment]
Found my problem. I'm guessing it had to do with changes to HomeKit in iOS 11.  I was using a temperature plugin for Homebridge that was doing a setCharacteristic for a serial number:

setCharacteristic(Characteristic.SerialNumber, this.options["serialnumber"])

This used to work fine but apparently Apple made changes that caused what I experienced above.  I just commented out the line, changed my Homebridge username again, and then everything was well.

Hopefully this helps someone else too.
